l10n_ch_hr_payroll/README.rst

```diff
@@ -1,31 +1,73 @@
-Swizerland Payroll Rules.
-=========================
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
 
-**Features list :**
```

Comment:
Please restore Feature list in here, this README.rst will be displayed as description of the module on Github, on Odoo instance and on apps.odoo.com